src/Engine/System/RenderSystem.cpp

```diff
@@ -7,18 +7,21 @@
 #include "ThirdParty/imgui/imgui_impl_glfw_gl3.h"
 #include "Scene/Scene.hpp"
 
-
-
 const Vector<DiffuseRenderComponent *> & RenderSystem::s_diffuseComponents(Scene::getComponents<DiffuseRenderComponent>());
 UnorderedMap<std::type_index, UniquePtr<Shader>> RenderSystem::s_shaders;
 const CameraComponent * RenderSystem::s_camera = nullptr;
+UniquePtr<SquareShader> RenderSystem::squareShader(
+    UniquePtr<SquareShader>::make("square_vert.glsl", "square_frag.glsl")
+);
 
 void RenderSystem::init() {
     glEnable(GL_DEPTH_TEST);
     glEnable(GL_CULL_FACE);
     glCullFace(GL_BACK);
     glEnable(GL_BLEND);
     glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+
+    squareShader->init();
```

Comment:
This should have similar error catching to what is found here:
https://github.com/jaafersheriff/BattleRoyale/blob/master/src/Engine/System/RenderSystem.hpp#L71

Reply:
Will do